napari/_qt/widgets/qt_layerlist.py

```diff
@@ -642,3 +646,8 @@ def _on_thumbnail_change(self, event=None):
             QImage.Format_RGBA8888,
         )
         self.thumbnailLabel.setPixmap(QPixmap.fromImage(image))
+
+    def close(self):
+        """Viewer is closing.
+        """
+        self.chunk_receiver.close()
```

Comment:
Made the change with parent above. But `QtChunkReciver.close()` disconnects two signals. Not sure if that's required? I assume I added it to fix a bug, but I don't remember. I think it might have been problem with running tests back-to-back? But can't be sure. If it's causing run-time problems is there a repo case?